### DISCUSSION TITLE
✅ KeyboardAccessoryViewLayout {ENTER} key?

### DISCUSSION THREAD

**Discussion by GuillaumeHullin** [isontheline/pro.webssh.net#1246] [Q&A] [answered]
Is there short-code for the "enter" key to put in my SSH Config File for `#!KeyboardAccessoryViewLayout`?

https://webssh.net/documentation/help/howtos/SSH/customise-keyboard-accessory-view-layout/

I would think it would be useful after pasting a snippet to be able to run it without the need of the full keyboard.

**Comment by isontheline** [comment]
Yet no way but you can add a carriage return to your snippet

Would you like a dedicated key anyway?

**Comment by GuillaumeHullin** [comment]
Actually both would be great.
- How do you add a carriage return in snippet? 
- a dedicated key would be great because I could call back last commands with up arrow and send it, without bring the all keyboard.

**Comment by isontheline** [comment]
> How do you add a carriage return in snippet?

Just add a new line at the end of snippet

> a dedicated key would be great because I could call back last commands with up arrow and send it, without bring the all keyboard.

No problem, I've just opened a new issue about your request : https://github.com/isontheline/pro.webssh.net/issues/1248
I will try to make it live asap

**Reply by isontheline** [reply, marked as answer]
Now available inside the latest beta release : https://testflight.apple.com/join/QSrBK59z
Documentation has been updated as well : https://webssh.net/documentation/help/howtos/SSH/customise-keyboard-accessory-view-layout/
